plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/VidarrPlugin.java

```diff
@@ -50,6 +54,99 @@
 
 public class VidarrPlugin extends JsonPluginFile<Configuration> {
 
+  private class ProvenanceCache extends ValueCache<Stream<VidarrProvenanceValue>> {
+    public ProvenanceCache(Path fileName) {
+      super("vidarr-provenance " + fileName.toString(), 30, ReplacingRecord::new);
+    }
+
+    private String formatListAsString(List<String> list) {
+      return list.stream().map(name -> ("\"" + name + "\"")).collect(Collectors.joining(","));
+    }
+
+    private String createVidarrProvenanceRequestBody(List<String> versionTypes) {
+      return "{"
+          + "\"analysisTypes\": [\"FILE\"],"
```

Comment:
Why the constraint of only reporting FILE? We don't really use URL right now but we might in the future.
Also, this constraint should be documented.

Reply:
I agree with Alexis about not constraining this, and it would be much easier to do so by updating a config file than having to do a Shesmu release.